model/marketplace_plugin.go

```diff
@@ -95,6 +96,7 @@ func (filter *MarketplacePluginFilter) ApplyToURL(u *url.URL) {
 	q.Add("build_enterprise_ready", strconv.FormatBool(filter.BuildEnterpriseReady))
 	q.Add("enterprise_plugins", strconv.FormatBool(filter.EnterprisePlugins))
 	q.Add("local_only", strconv.FormatBool(filter.LocalOnly))
+	q.Add("arch", filter.Architecture)
```

Comment:
Would we consider sending up `OS` and `Arch` discretely instead?

Reply:
unless we call it something like `platform`?